portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -30,17 +32,19 @@ definition {
 	@priority = "4"
 	@refactorneeded
 	test AddBlogsEntryWithImageViaBlogsImages {
-		ProductMenu.gotoPortlet(
-			category = "Site Builder",
-			portlet = "Pages");
-
-		SitePages.addPublicPage(pageName = "${pageName}");
-
-		Navigator.gotoPage(pageName = "${pageName}");
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Widget Page Name");
```

Comment:
Update the groupName
Make the layoutName unique to `WYSWIYG Page`

Same for below.